strongbox-storage/strongbox-storage-api/pom.xml

```diff
@@ -288,6 +288,13 @@
             <artifactId>jackson-jaxrs-json-provider</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.javatuples</groupId>
+            <artifactId>javatuples</artifactId>
+            <version>1.2</version>
```

Comment:
No version is required here, since it's defined in the parent.

Reply:
see below